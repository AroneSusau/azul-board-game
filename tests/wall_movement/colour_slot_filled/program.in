2
tests/wall_movement/colour_slot_filled/testing.azul
turn 1 R 1
