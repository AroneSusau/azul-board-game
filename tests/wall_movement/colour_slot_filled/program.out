[H[2J
[34m       db[35m   8888P[31m 8    8[33m 8    
[34m     dPYb[35m    dP  [31m8    8[33m 8     
[34m   dPwwYb[35m  dP   [31m8b..d8[33m 8      
[34m dP    Yb[35m d8888 [31m`Y88P'[33m 8888   
[0m
Enter a number
[37m1.[2m Start Game[0m
[37m2.[2m Load Game[0m
[37m3.[2m Credits[0m
[37m4.[2m Exit[0m
[0m[32m> [0m[H[2JEnter the filename from which load a game
[32m> [0m[32mAzul game successfully loaded[0m

[2mPress enter to continue..[0m[H[2J[43;1m[30mLets play!
[0m
=== Help Menu ===
COMMAND:       EXAMPLE                             DESCRIPTION

turn:          turn <factory> <colour> <pile>      Player turns, moving tiles from factory to the player pile line.
discard:       discard <factory> <colour>          Player turns, moving tiles from factory to the player discard pile.
save:          save <filename>                     Saves the game to the specified filename.
help:          help                                Prints this help menu.
exit / ^D:     exit                                Exits the game.

[2mPress enter to continue..[0m[H[2JTURN FOR PLAYER: b

Factories:
0: F 
1: R U L Y 
2: U U L B 
3: B U R R 
4: B L Y R 
5: R L L R 

Mosaic for a                        [47m[30mMosaic for b[0m
1:         . || [47m[30m . [0m[47m[30m . [0m[47m[30m . [0m[47m[30m . [0m[47m[30m . [0m     1:         . || [41;1m R [0m[43;1m[30m Y [0m[44;1m B [0m[46;1m L [0m[40;1m U [0m
2:       . . || [47m[30m . [0m[47m[30m . [0m[47m[30m . [0m[47m[30m . [0m[47m[30m . [0m     2:       . . || [47m[30m . [0m[47m[30m . [0m[47m[30m . [0m[47m[30m . [0m[47m[30m . [0m
3:     . . . || [47m[30m . [0m[47m[30m . [0m[47m[30m . [0m[47m[30m . [0m[47m[30m . [0m     3:     . . . || [47m[30m . [0m[47m[30m . [0m[47m[30m . [0m[47m[30m . [0m[47m[30m . [0m
4:   . . . . || [47m[30m . [0m[47m[30m . [0m[47m[30m . [0m[47m[30m . [0m[47m[30m . [0m     4:   . . . . || [47m[30m . [0m[47m[30m . [0m[47m[30m . [0m[47m[30m . [0m[47m[30m . [0m
5: . . . . . || [47m[30m . [0m[47m[30m . [0m[47m[30m . [0m[47m[30m . [0m[47m[30m . [0m     5: . . . . . || [47m[30m . [0m[47m[30m . [0m[47m[30m . [0m[47m[30m . [0m[47m[30m . [0m
broken: . . . . . . .               broken: . . . . . . . 
[32m> [0m
[31mError: Invalid input.[0m

[2mPress enter to continue..[0m[H[2J[H[2J[H[2J[H[2J
[32m __                      _        _                
[32m //_ _  _  /_ /_/_      /_`_  _  /_//_     ._  _  /
[32m// //_|/ //\  / /_//_/ /  /_//  /  //_|/_/// //_/. 
[32m                                       _/     _/   
[0m
[0m